templates/profile.j2

```diff
@@ -25,12 +25,10 @@ if [ ! -z $JAVA_HOME ]; then
     JAVA_CMD=$JAVA_HOME/bin/java
 fi
 
-
-
 export CLI_CP=$(find /var/lib/rundeck/cli -name \*.jar -printf %p:)
 export BOOTSTRAP_CP=$(find /var/lib/rundeck/bootstrap -name \*.jar -printf %p:)
 export RDECK_JVM="-Djava.security.auth.login.config=/etc/rundeck/jaas-loginmodule.conf \
-       -Dloginmodule.name=RDpropertyfilelogin \
+       -Dloginmodule.name={{ 'ldap' if rd_login_module == 'ldap' else 'RDpropertyfilelogin' }} \
```

Comment:
This conditional check could be improved. http://jinja.pocoo.org/docs/2.9/templates/#if